components/popup/src/vwc-popup-base.ts

```diff
@@ -0,0 +1,198 @@
+import {
+	PropertyValues, html, LitElement, property, query, TemplateResult
+} from 'lit-element';
+import { ClassInfo, classMap } from 'lit-html/directives/class-map.js';
+import { nothing } from 'lit-html';
+import { computePosition, offset, shift, flip, arrow } from '@floating-ui/dom';
+import type { Placement, Strategy, Padding } from '@floating-ui/core';
+import '@vonage/vwc-elevation';
+
+export class VWCPopupBase extends LitElement {
+	@query('.popup') protected popupEl!: HTMLElement;
+	@query('.popup-arrow') protected arrowEl!: HTMLElement;
+	protected padding: Padding = 6;
+
+	/**
+	 * @prop open - indicates whether the popup is open
+	 * accepts boolean value
+	 * @public
+	 * */
+	@property({ type: Boolean, reflect: true })
+		open?: boolean= false;
```

Comment:
If it is false by default, why the optional sign (`?`)?

Reply:
right, it should even throw on that.
annotation is redundant as well
```suggestion
		open = false;
```